app/uk/gov/hmrc/gvmshaulierstatus/repositories/HaulierStatusRepository.scala

```diff
@@ -30,24 +31,28 @@ import uk.gov.hmrc.play.http.logging.Mdc
 
 import java.time.{Clock, Instant}
 import javax.inject.{Inject, Singleton}
+import scala.concurrent.duration.SECONDS
 import scala.concurrent.{ExecutionContext, Future}
 
 @SuppressWarnings(Array("org.wartremover.warts.Any"))
 @Singleton
 class HaulierStatusRepository @Inject()(
-  mongo:                     MongoComponent
-)(implicit executionContext: ExecutionContext)
+  mongo:                     MongoComponent,
+)(implicit executionContext: ExecutionContext, appConfig: AppConfig)
     extends PlayMongoRepository[HaulierStatusDocument](
       collectionName = "haulier-status",
       mongoComponent = mongo,
       domainFormat   = HaulierStatusDocument.mongoFormat,
       indexes = Seq(
         IndexModel(ascending("id"), IndexOptions().name("correlationId").unique(true).sparse(true)),
-        IndexModel(ascending("dateTime"), IndexOptions().name("dateTime").sparse(false)),
+        IndexModel(
+          ascending("dateTime"),
```

Comment:
we normally name this field as lastUpdated , any reason why this named as dateTime ?  if no specific reason, I can name this lastUpdated

Reply:
Agree `dateTime` has no semantic value. Can we go with `creationTime` rather than `lastUpdated`, as that's what the field represents? (ie. we're not going to be updating this particular field with a new value).

Reply:
changed to `createdAt`